### ISSUE TITLE
False warning for subscriptions with single level wildcards

### ISSUE THREAD

**Issue by hagor** [christianTF/LoxBerry-Plugin-MQTT-Gateway#52]
The log shows warnings for topics which supposedly do not match any subscribed topics altough the topics are subscribed with a single level wildcard (+)

**Example:**
Subscription Topic: tele/+/SENSOR
Log Message:
![image](https://user-images.githubusercontent.com/12829879/92885726-58d2fe00-f413-11ea-9638-10b4f08f594e.png)

Anyway the data gets correctly forwarded to the virtual input of the miniserver.

**Comment by christianTF** [comment]
Is this version 1.1.2?

**Comment by hagor** [comment]
yes, latest release 1.1.2

**Comment by christianTF** [comment]
I have to finish something in LoxBerry-Core first, then I will test and reproduce this issue.

**Comment by christianTF** [comment]
Reproduced:
Subscription: 
`tele/+/SENSOR`
Payload:
`mosquitto_pub -u loxberry -P mypassword -t tele/Wohnzimmer/SENSOR -m "Hallo"`

Log Messages:
```
19:21:35.093 <INFO> MQTT received: tele/Wohnzimmer/SENSOR: Hallo
19:21:35.095    tele/Wohnzimmer/SENSOR is cached
19:21:35.097 <WARNING> Incoming topic does not match any subscribed topic. This might be a bug
19:21:35.098 <WARNING> Topic: tele/Wohnzimmer/SENSOR
```

**Comment by christianTF** [comment]
Should be fixed with https://github.com/christianTF/LoxBerry-Plugin-MQTT-Gateway/commit/aaccba27aa30957338f0b0578d9d3896dae65718

**Comment by christianTF** [comment]
@hagor Could you please install the current master https://github.com/christianTF/LoxBerry-Plugin-MQTT-Gateway/archive/master.zip and test your use-case again.
Please also take a look at all subscriptions, there should be no warning 'does not match nay subscriped topic' with any topic.

It is quite annoying to create a regex by regex :upside_down_face:

Thanks, Christian

**Comment by hagor** [comment]
I can confirm, the current master fixed the issue, no more warnings.

Somehow I must think of the saying "Some people think they can solve a problem with a regex. Now they have two problems" ;-)

Thanks for the fix! 👍

**Comment by christianTF** [comment]
Fixed with aaccba2
